specification/resources/resource-manager/Microsoft.Management/2017-11-01-preview/management.json

```diff
@@ -0,0 +1,673 @@
+      "get": {
+        "tags": [
+          "ManagementGroups"
+        ],
+        "operationId": "ManagementGroups_List",
+        "description": "List management groups for the authenticated user.\n",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "$skiptoken",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Page continuation token is only used if a previous operation returned a partial result. \nIf a previous response contains a nextLink element, the value of the nextLink element will include a token parameter that specifies a starting point to use for subsequent calls.\n"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ManagementGroupListResult"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "GetManagementGroups": {
+            "$ref": "./examples/listManagementGroups.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/managementGroups/{groupId}": {
+      "get": {
+        "tags": [
+          "ManagementGroups"
+        ],
+        "operationId": "ManagementGroups_Get",
+        "description": "Get the details of the management group.\n",
+        "parameters": [
+          {
+            "$ref": "#/parameters/GroupIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ExpandParameter"
+          },
+          {
+            "$ref": "#/parameters/RecurseParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ManagementGroup"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetManagementGroup": {
+            "$ref": "./examples/getManagementGroup.json"
+          },
+          "GetManagementGroupWithExpand": {
+            "$ref": "./examples/getManagementGroupWithExpand.json"
+          },
+          "GetManagementGroupsWithExpandAndResurse": {
+            "$ref": "./examples/getManagementGroupWithExpandAndRecurse.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ManagementGroups"
+        ],
+        "operationId": "ManagementGroups_Create",
```

Comment:
For methods like this the convention is to name them CreateOrUpdate.

Reply:
Fixed.